### PR TITLE
[FIX] deprecated function use

### PR DESCRIPTION
**Pull request by atomatis**
Fixes php-school/cli-menu#249

### CODE REVIEW

File: src/MenuStyle.php [hunk 1]
```diff
@@ -300,11 +302,11 @@ public function getDisabledItemText(string $text) : string
     /**
      * Get the ansi escape sequence for the foreground colour.
      *
-     * @param bool $bright Whether to modify to the ansi bright variation
+     * @param bool|null $bright Whether to modify to the ansi bright variation
      *
      * @return string
      */
-    private function getForegroundColourCode(bool $bright = false) : string
+    private function getForegroundColourCode(?bool $bright = false) : string
```
Comment (AydinHassan) [inline src/MenuStyle.php:309]: Why is this changed? It doesn't seem related

File: src/MenuStyle.php [hunk 2]
```diff
@@ -509,8 +511,8 @@ private function generatePaddingTopBottomRows() : void
             );
         }
 
-        $this->paddingTopBottom = $this->paddingTopBottom >= 0 ? $this->paddingTopBottom : 0;
```
Comment (AydinHassan) [inline src/MenuStyle.php:512]: I'm confused how this even becomes null

**Comment by AydinHassan** [comment]
Hi @atomatis - thanks for the PR! If you don't mind, i'd like to do a little debugging tomorrow to see if we are fixing this in the correct place

**Comment by AydinHassan** [comment]
hey @atomatis - while we really appreciate the PR - we're going to go with a different fix (more at the root) with #252 - sorry for wasting your time and hope you understand!

**Comment by atomatis** [comment]
Hi, thank you for reactivity.
Sorry for my Clumsy PR ^^'

It's ok. That will train me to do better next time :)

**Comment by AydinHassan** [comment]
It's not clumsy at all, it pointed me in the right direction so thanks for your help!
